variablegroups/scripts/get-all-variable-groups.sh

```diff
@@ -10,11 +10,11 @@ fi
 ACCESSTOKEN=$1
 USERNAME=${2:azuredevops}
 
-groupList=$(curl -X GET -s -u ${USERNAME}:${ACCESSTOKEN} "https://dev.azure.com/hmcts/cnp/_apis/distributedtask/variablegroups/?api-version=5.0-preview.1" | jq -r '.value[] | .name' )
+groupList=$(curl -X GET -s -u ${USERNAME}:${ACCESSTOKEN} "https://dev.azure.com/hmcts/cnp/_apis/distributedtask/variablegroups/?api-version=5.0-preview.1" | jq -r '.value[] | .name  | if contains("AKS") then . else empty end' )
 
 
 for group in ${groupList} ; do
 
-curl -X GET -s -u ${USERNAME}:${ACCESSTOKEN} "https://dev.azure.com/hmcts/cnp/_apis/distributedtask/variablegroups/?groupName=${group}&api-version=5.0-preview.1" | jq -r '.value[0] | del(.createdOn, .modifiedBy , .createdBy, .modifiedOn, .providerData.lastRefreshedOn)' > ${group}.json
```

Comment:
this didn't work?

Reply:
yes, this doesn't work . It leaves it in non editable stage.